addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -76,7 +76,7 @@ const TableOfContent = publicWidget.Widget.extend({
      * @override
```

Comment:
Commit message: 
Maybe add [this task](https://www.odoo.com/odoo/974/tasks/4160033) in your PR, as it seems to be the same issue :wink: (your fix solved them too)

Maybe the second issue will help you investigate, as it happens with only one snippet so it will be easier to find what is happening :slightly_smiling_face: